src/components/search-component/index.js

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react";
+import teamdata from "./../card-component/config";
+
+const SearchComponent = ({ setsearchfiltertext }) => {
+  const [searchval, setsearchval] = useState("");
+  //function to submit search value
+  const submitsearch = (e) => {
+    e.preventDefault();
+    setsearchfiltertext(filtersearchdata(searchval));
+  };
+  // function return arry of filter out data
+  const filtersearchdata = (searchval) => {
+    return teamdata.filter((search) => {
```

Comment:
Try to make this a utility/reusable function.

Reply:
Thank you for comment.
sure, will make it common if any use case will appear in future.